### ISSUE TITLE
MidasDepthEstimationWebcam example throws error

### ISSUE THREAD

**Issue by brad-miller** [cansik/deep-vision-processing#8]
Hi Florian
Thank for your update to deepvision (0.6.0) library

Whe using  Processing 4.0.3Alpha the MidasDepthEstimationWebcam example throws:
RuntimeException: resize() not implemented for this PImage type
using Processing video library using GStreamer 1.16.2

works with Processing 3.5.4

kind regards

**Comment by cansik** [comment]
As far as I know they reimplemented the `PImage` class to remove dependencies to AWT. And not all methods have been implemented yet. This breaks a lot of sketches.

What we could do is resize the final image with OpenCV instead of Processing. Will have a look later.

**Comment by cansik** [comment]
Here the information from [Processing4](https://github.com/processing/processing4/blob/master/core/src/processing/core/PImage.java#L484):

```java
/**
 *
 * Resize the image to a new width and height. To make the image scale
 * proportionally, use 0 as the value for the <b>wide</b> or <b>high</b>
 * parameter. For instance, to make the width of an image 150 pixels, and
 * change the height using the same proportion, use resize(150, 0).<br />
 * <br />
 * Even though a PGraphics is technically a PImage, it is not possible to
 * rescale the image data found in a PGraphics. (It's simply not possible
 * to do this consistently across renderers: technically infeasible with
 * P3D, or what would it even do with PDF?) If you want to resize PGraphics
 * content, first get a copy of its image data using the <b>get()</b>
 * method, and call <b>resize()</b> on the PImage that is returned.
 *
 * @webref pimage:method
 * @webBrief Resize the image to a new width and height.
 * @usage web_application
 * @param w the resized image width
 * @param h the resized image height
 * @see PImage#get(int, int, int, int)
 */
public void resize(int w, int h) {  // ignore
  throw new RuntimeException("resize() not implemented for this PImage type");
}
```

**Comment by cansik** [comment]
I fixed it in release [0.6.1](https://github.com/cansik/deep-vision-processing/releases/tag/v0.6.1-alpha).
Would be great if you could test it and give a quick feedback.

**Comment by brad-miller** [comment]
Issue resolved

Running on mbp 15” (mid 2017) 2.8 GHz Quad-Core Intel Core i7

MacOS 10.15.7

Processing 4.0.3 Alpha

CPU only performance is 2fps

Kind regards

brad

On 24 Feb 2021, at 4:34 am, Florian Bruggisser <notifications@github.com<mailto:notifications@github.com>> wrote:



I fixed it in release 0.6.1<https://github.com/cansik/deep-vision-processing/releases/tag/v0.6.1-alpha>.
Would be great if you could test it and give a quick feedback.

**Comment by cansik** [comment]
Ok great, I am working on integrating [Midas 2.1 small](https://github.com/intel-isl/MiDaS) which should run with 8 FPS at least. But that's a future topic.
